### ISSUE TITLE
Issue without description not showing up

### ISSUE THREAD

**Issue by rcastillo2589** [bspride/jiraMenuBar#1]
When clicking on an issue that does not contain a description, the issue component fails to render the entire page.

**Comment by bspride** [comment]
Ahh i bet because the template fails out. Should be able to make the variable in the template optional and fix it.

**Comment by rcastillo2589** [comment]
Yeah, that's the reason. I'm going to set a condition and if the description is provided set it, else set a default value something like 'No description'

**Comment by bspride** [comment]
yea a simple *ngIf would take care of that
